### ISSUE TITLE
update goldchain client to make it work against the latest alpha release

### ISSUE THREAD

**Issue by GlenDC** [threefoldtech/jumpscaleX_libs#26]
Goldchain now has custody fees which have to be paid for each coin input spent. The JSX GoldChain client will have to be updated to support this.

Coin Destruction Txs (minting extension) now also accept multiple coin outputs. This has to be adapted as well.

**Comment by GlenDC** [comment]
Goldchain is now supported once again, and some other improvements that were added in the Desktop Wallet have also been ported to this goldchain JSX client.

Some snippets to show the custody fee support:

```
JSX> w.balance                                                                                                                                                         
wallet balance on height 4971 at 2019/10/17 11:37:12
300 GFT available and 0 GFT locked
Custody Fee to be paid: 0 GFT
```

```
JSX> w.coins_send('016a2cd19cdc573c83d472fd7361a2a67aa68de7776031f79b93dffc6b6091c1de035d023d8505', '42 GFT')                                                          
TransactionSendResult(transaction=transaction v1 442e9fef0af90720f4200de18e2f3e085fab2bb7ac60c78862b39221dc8fa521, submitted=True)
```

```
JSX> w.balance                                                                                                                                                         
wallet balance on height 4971 at 2019/10/17 11:37:12
0 GFT available and 0 GFT locked
Custody Fee to be paid: 0 GFT
Unconfirmed: 299.999 GFT available 0 GFT locked
Unconfirmed Balance Deduction: -300 GFT
```

```
JSX> w.balance                                                                                                                                                         
wallet balance on height 4983 at 2019/10/17 12:05:09
299.998909202 GFT available and 0 GFT locked
Custody Fee to be paid: 0.000090798 GFT
```

**Comment by GlenDC** [comment]
Resolved by d461cc4cf12c4069b4b05a3c5ae5d39a39cd5df9.
